### ISSUE TITLE
"This app can't run on your PC."

### ISSUE THREAD

**Issue by patoxine** [TeamTernate/infinite-music-discs#109]
I keep getting the "This app can't run on your PC. To find a version for your PC, check with the software publisher." error every time I press the "Generate" button. I'm using windows 10. Is there a way to fix this?

**Comment by bradytheinventor** [comment]
Yes, your computer must not like the parallel processing that was added in v1.6. Turn on the "Convert tracks to .ogg one at a time" setting to get rid of that error. It will take longer to generate your pack though because it's only processing one track at a time instead of several.

**Comment by Asera2** [comment]
I am having the same issue.
Unfortunately turning on that option does not help, the error still pops up

I am using Windows 10, tried opening it as admin, tried compatibility mode, nothing helped so far

**Comment by N3XUSEXE** [comment]
I've tried all the advice from above, even converting the files to .OGG myself (using an online converter, dunno if that's the issue though) and still, every time I try to generate a pack, it gives me the "This app cannot run on your PC." error.
I was using a workaround with the old python script, but it only worked the first time. When I decided I wanted more music, the pack broke and every new generation had the same issue.
I am really hoping that a solution to this issue is found because this would be really useful to me for projects I am currently working on.
